<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>535</width>
        <height>552</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Miscellaneous</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkbox_immediately_rename">
            <property name="text">
             <string>Offer to rename new items immediately</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="checkbox_autoresponse">
            <property name="text">
             <string>Enable auto-response</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QCheckBox" name="checkbox_toolbar_text">
            <property name="text">
             <string>Show text in toolbar</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="checkbox_show_random_tips">
            <property name="text">
             <string>Show random tips on start-up</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Backups</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinbox_autosave_interval">
            <property name="suffix">
             <string> minute(s)</string>
            </property>
            <property name="prefix">
             <string>every </string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_autosave_interval">
            <property name="text">
             <string>Auto-save interval:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkbox_enable_autosave">
            <property name="text">
             <string>Automatically create backups</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="button_browse_autosave">
            <property name="text">
             <string>Open backup folder</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Updates</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkbox_auto_update_check">
            <property name="text">
             <string>Check for updates on start-up</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="button_update_check">
            <property name="text">
             <string>Check for updates now</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/update.png</normaloff>:/icons/update.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Run options</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkbox_opensesamerun">
            <property name="text">
             <string>Run experiments in a separate process</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="checkbox_auto_opensesamerun_exec">
            <property name="text">
             <string>Auto-detect the run command</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_opensesamerun_exec">
            <property name="text">
             <string>Custom run command:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="edit_opensesamerun_exec"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkbox_opensesamerun</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkbox_auto_opensesamerun_exec</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>522</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkbox_auto_opensesamerun_exec</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_opensesamerun_exec</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkbox_auto_opensesamerun_exec</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_opensesamerun_exec</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkbox_auto_opensesamerun_exec</sender>
   <signal>toggled(bool)</signal>
   <receiver>edit_opensesamerun_exec</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkbox_enable_autosave</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinbox_autosave_interval</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkbox_enable_autosave</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_autosave_interval</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
